koku/api/dump/models.py

```diff
@@ -0,0 +1,48 @@
+"""Models for user-initiated data dump requests."""
+
+from uuid import uuid4
+
+from django.db import models
+
+
+class DumpRequest(models.Model):
```

Comment:
It's rare that I comment on naming, but maybe we can call all everything related to pulling raw data out of the database a DataExport instead of Dump. I think the user experience there is a little better.

Reply:
"dump" has been the term of choice since we were originally talking about just getting data via `pg_dump`, but I understand the hesitation on language…

💩

Reply:
The classes have been renamed in the latest push to `DataExportRequest`*.